### DISCUSSION TITLE
Localize modern template

### DISCUSSION THREAD

**Discussion by mikasoukhov** [dotnet/docfx#9797] [Q&A]
Any options make the subj without cloning the template locally? It is possible to override token.json file only?

**Comment by filzrev** [comment]
> It is possible to override token.json file only?

As far as I knows. There is no such options available that override `token.json` file only.
It need to creating docfx `custom template` that replace `token.json` file.

### Steps to replace `token.json` file
**1. Modify `docfx.json` config file to use `custom` templates**

```
    "template": [
      "default",
      "modern",
      "templates/custom"
    ],
```

**2. Create `token.json` file at  `templates/custom`  directory with localized contents.**
